canonical-web-and-design/anbox-cloud.io
# Test add-streaming-sdk

## Done

- [x] Implementation of the Streaming SDK #17 using example parameters (max resolution available on landscape mode)
- [x] Unit and mocked integration tests

## QA:

- Check out this feature branch
- Run the site using the command `./run serve`
- Turn on the VPN
- View the site locally in your web browser at: http://0.0.0.0:8043/demo
- If you don't have access you need to ask for an invitation code and hit http://0.0.0.0:8043/login?invitation_code=XXXXX&next=/demo where XXXXX is where your invitation code goes.
- Once you log in, you should see the demo "Big Buggy Racing" game loading

### A few considerations:
- The game may take some time to load. If it doesn't eventually load, you'd need to refresh the page.
- Current configuration uses keyboard instead of touch. Use arrow keys to control the buggy.
- Sometimes, it might be difficult to play the game, as there is some delay.

## Issue / Cards:

- Tested #17  
- On this PR, an instance of the SDK was created and the example parameters were added #18
